dwh_design_conventions.md

```diff
@@ -110,14 +110,14 @@ staging:
 
 ### Marts of Facts

```

Comment:
We should add screenshots of how the directories should look. We can use FTA's internal analytics?